src/components/ActiveClients.js

```diff
@@ -105,9 +105,8 @@ const ActiveClients = () => {
             <li>
               <strong>Search</strong> by: Debug id, OS, Application, and Geo.
             </li>
-            <li>
-              Click on a debug tag to see (up to) the last <strong>100 pings</strong>.
-            </li>
+            <li>Click on <strong>Pings</strong> to see (up to) the last 100 pings for a Debug Tag.</li>
+            <li>Click on <strong>Event Stream</strong> to see an aggregated timeline for all events in all pings for a Debug Tag.</li>
```

Comment:
Is this still the "last 100 pings" when you say "all pings" here? Might want to clarify that, I imagine this is something someone would ask.